lib/state/actions/pokemon_list_actions.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex/state/app_state.dart';
+
+class GetPokemonList extends ReduxAction<AppState> {
```

Comment:
For action naming we normally add "Action" word as its suffix therefore maybe you could rename this to `GetPokemonsAction` rather? 🙂 

also, for your action.dart file instead of having a specific name (`pokemon_list_action.dart`), kindly make it a general one rather => `pokemon_actions.dart`